### PR TITLE
Community mod activity

### PR DESCRIPTION
**Pull request by nilllzz**

### CODE REVIEW

File: src/_common/community/activity-item/activity-item.vue [hunk 1]
```diff
@@ -0,0 +1,260 @@
...
+						'-icon-other': color !== 'notice' && color !== 'theme',
+					}"
+				>
+					<app-jolticon v-if="shouldShowIcon" :icon="icon" />
+				</span>
+
+				<div class="-action">
+					<span v-if="item.type === 'community/created'" v-translate>
+						<em>Created</em> this awesome community.
+					</span>
```
Comment (YLivay) [inline src/_common/community/activity-item/activity-item.vue:54]: I thought we were phasing away v-translates. do we have a plan for sanitizing these?

Comment (nilllzz) [reply src/_common/community/activity-item/activity-item.vue:54]: Oh I wasn't aware, why are we not doing v-translate?
Should we just do <translate> tags instead?

And what do you mean with "sanitize"?

File: src/_common/community/activity-item/activity-item.vue [hunk 2]
```diff
@@ -0,0 +1,260 @@
...
+						</span>
+						<span
+							v-if="isToday || isYesterday"
+							v-translate="{ time: date(item.added_on, 'shortTime') }"
+						>
+							at %{ time }
+						</span>
+						<span v-else>
+							{{ date(item.added_on, 'shortDate') }}
+						</span>
```
[the remark this answers is not in the document]
Comment (nilllzz) [reply src/_common/community/activity-item/activity-item.vue:35]: This seemed a little more natural for this type of feed.

File: src/app/views/communities/view/edit/activity/activity.ts [hunk 3]
```diff
@@ -0,0 +1,122 @@
...
+				this.items.length === 0 ||
+				item.type === CommunityActivityItem.TYPE_COMMUNITY_CREATED
+			) {
+				// The first item will always be a user/time split.
+				newItem.timesplit = true;
+				newItem.usersplit = true;
+			} else {
+				// Compare to the last item in the list.
+				// When it's a different day, have a full split (user and time).
+				// When the difference in time is more than 20 minutes or it's a different user, have a user split.
```
Comment (hapurubokka) [inline src/app/views/communities/view/edit/activity/activity.ts:94]: I believe it's currently set up for 30 minutes, on line 100.

**Pull request merged**